Resources/Prototypes/_Goobstation/GameRules/Dynamic/dynamic.yml

```
@@ -7,19 +7,15 @@
 - type: dataset # Funky station unfucked dynamic
   id: DynamicRoundstartGamerules
   values:
-  - TraitorDynamicLow # traitors
-  - TraitorDynamicMedium
-  - TraitorDynamicHigh
-  - ThiefDynamic # thieves (spy todo)
-  - ChangelingDynamicLow # changelings
-  - ChangelingDynamicHigh
-  - HereticDynamicLow # heretics
-  - HereticDynamicHigh
+  - TraitorDynamic # traitors
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)